regression-tests.dnsdist/test_Advanced.py

```diff
@@ -1595,3 +1596,25 @@ def testTempFailureTTLBinding(self):
         receivedQuery.id = query.id
         self.assertEquals(query, receivedQuery)
         self.assertEquals(receivedResponse, response)
+
+class TestAdvancedEDNSOptionRule(DNSDistTest):
+
+    _config_template = """
+    newServer{address="127.0.0.1:%s"}
+    addAction(EDNSOptionRule(8), DropAction())
+    """
+
+    def testDropped(self):
```

Comment:
It would be nice to add a test proving that a query without the corresponding option is not dropped :)

Reply:
pushed